server/polar/file/schemas.py

```diff
@@ -26,6 +26,12 @@ class DownloadableFileCreate(FileCreateBase):
     """Schema to create a file to be associated with the downloadables benefit."""
 
     service: Literal[FileServiceTypes.downloadable]
+    size: int = Field(
```

Comment:
Why is this attribute needed? If you add it here, you should provide a value in the model if not you cannot download it.